src/pac.c

```diff
@@ -112,10 +112,24 @@ pac_may_have_muzzy(pac_t *pac) {
 	return pac_decay_ms_get(pac, extent_state_muzzy) != 0;
 }
 
+size_t pac_batched_alloc_retained_size(size_t size) {
+	if (size > SC_LARGE_MAXCLASS) {
+		return size;
```

Comment:
Return 0 for overflow, which is consistent with sz_s2u_compute

Reply:
With `sz_s2u` guarding for us, any request reaching here should be valid, so we shouldn't return 0 for it even the size is larger than `SC_LARE_MAXCLASS`. So the question should be how can a valid reuqest reach here with `size > SC_LARGE_MAXCLASS`?

If a request with usize `SC_LARGE_MAXCLASS` is passed in, adding `sz_large_pad`, `size` could exceed the limit. Since it is still valid, we should not return 0. Since it cannot further grow due to the assertion [here](https://github.com/jemalloc/jemalloc/blob/499f3068593ec61dae961e2c8ea3e0cf1482d616/src/sz.c#L37), it cannot be ceiled to higher value as well. Thus, we should return size directly. Added an explanation in the comment.